packages/data/addon/models/navi-dimension.ts

```diff
@@ -30,17 +30,13 @@ export default class NaviDimensionModel extends NativeWithCreate {
   /**
    * Dimension value
    */
-  declare suggestions?: Readonly<unknown[]>;
+  declare suggestions?: Readonly<Record<string, string>>;
 
   /**
    * Dimension value for display purposes
    */
   get displayValue() {
-    const { value, suggestions } = this;
-    if (suggestions?.length) {
-      return `${value} (${suggestions.join(', ')})`;
-    }
-    return `${value}`;
+    return `${this.value}`;
```

Comment:
The dimension-select component expects this to contain the suggestions so that it performs the local search correctly. Can we try to add a test for that to show searching by suggestion doesn't work for small cardinality and then add a fix there to explicitly search the suggestions